Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -195,25 +197,10 @@ internal void SetClientOwner(NetworkConnection conn)
         /// </summary>
         public static void ResetNextNetworkId() => nextNetworkId = 1;
 
-        /// <summary>
-        /// The delegate type for the clientAuthorityCallback.
-        /// </summary>
-        /// <param name="conn">The network connection that is gaining or losing authority.</param>
-        /// <param name="identity">The object whose client authority status is being changed.</param>
-        /// <param name="authorityState">The new state of client authority of the object for the connection.</param>
-        public delegate void ClientAuthorityCallback(NetworkConnection conn, NetworkIdentity identity, bool authorityState);
-
-        /// <summary>
-        /// A callback that can be populated to be notified when the client-authority state of objects changes.
-        /// <para>Whenever an object is spawned using SpawnWithClientAuthority, or the client authority status of an object is changed with AssignClientAuthority or RemoveClientAuthority, then this callback will be invoked.</para>
-        /// <para>This callback is only invoked on the server.</para>
-        /// </summary>
-        public static ClientAuthorityCallback clientAuthorityCallback;
-
```

Comment:
@paulpach I think you need to rebase the PR. or did you mean to remove clientAuthorityCallback?

Reply:
fixed